facebook/OpenBIC
# common: shell: Add shell ipmi command

Summary:
- Add bic ipmi command for self testing which is including **raw command** and **command scanning**.

Test Plan:
- Build Code: PASS
- Check shell command in BIC console: PASS

Log:
- BIC console:
  ```
  uart:~$ platform ipmi scan 
  ------------------------------------------------------------------------------
  * netfn 0h:
  00000000: 01                                               |.                |
  * netfn 4h:
  00000000: 2d                                               |-                |
  * netfn 6h:
  00000000: 01 02 03 04 37 52                                |....7R           |
  * netfn ah:
  00000000: 10 11 12 22 23 44                                |..."#D           |
  * netfn 30h:
  00000000: cb e2 ef f0                                      |....             |
  * netfn 38h:
  00000000: 01 02 03 09 0a 0b 12 14  15 16 1f 21 22 23 25 28 |........ ...!"#%(|
  00000010: 29 30 31 32 41 45 60 64  66 68 69 88 a0 b0       |)012AE`d fhi...  |
  ------------------------------------------------------------------------------

  uart:~$ platform ipmi raw 0x06 0x01
  BIC self command netfn:0x6 cmd:0x1 response with cc 0x0
  00000000: 00 80 01 01 02 bf 15 a0  00 00 00 00 00 00 00    |........ ....... |
  
  uart:~$ platform ipmi raw 0x38 0x0a 0x15 0xa0 0x00 0x01
  BIC self command netfn:0x38 cmd:0xa response with cc 0x0
  00000000: 15 a0 00 57 61 69 6d 65  61 20 43 61 6e 79 6f 6e |...Waime a Canyon|
  ```
